Geometry/HGCalCommonData/src/HGCalGeomParameters.cc

```diff
@@ -1065,10 +1080,45 @@ void HGCalGeomParameters::loadSpecParsHexagon8(const cms::DDFilteredView& fv,
     }
   }
 
+  // Read in parameters from Philip's file
+  if (php.waferMaskMode_ > 1) {
```

Comment:
there are a lot of conditionals and comparisons here and in the similar constructs below.

does this noticeably increase the geometry initialization time?

Reply:
Changed